packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -210,12 +210,30 @@
 							:chart-config="{ selectedRun: 'fixme', selectedVariable: selectedVariables }"
 							:multi-select="true"
 							:show-remove-button="false"
-							:variables="Object.keys(pyciemssMap).filter((c) => modelPartTypesMap[c] !== 'parameter')"
+							:variables="Object.keys(pyciemssMap).filter((c) => modelPartTypesMap[c] === 'state')"
```

Comment:
Why are we adding this filter?
If I am understanding correctly as a user I will no longer be able to check observables in this chart?

Reply:
We don't want to show all the unnecessary variables for selection for this dropdown list for the variable chart. We want to show the only state variables.

Reply:
Although looking at a parameter like beta may be better seen on the chart above this one 
I think that seeing observables like hospitalized is worth allowing the user to do here
Previously we were filtering not Param, if you want to be explicit with what we are (instead of are not) allowing i think we should allow `state` as well as `observable`

https://github.com/DARPA-ASKEM/terarium/blob/d415c37efe59f1908c2e4289f94b45cd6c10f551/packages/client/hmi-client/src/services/model.ts#L195-L201

Reply:
Ok I just added observables to the list.